project/Dependencies.scala

```diff
@@ -60,7 +60,7 @@ object Dependencies {
     val sigar         = "org.fusesource"              % "sigar"                        % "1.6.4"       // ApacheV2
 
     // reactive streams
-    val reactiveStreams = "org.reactivestreams"       % "reactive-streams"             % "1.0.0" // CC0
+    val reactiveStreams = "org.reactivestreams"       % "reactive-streams"             % "1.0.0" // CC0 // TODO update to 1.0.1 once it's released, we don't want to ship artifacts depending on RCs
```

Comment:
notice that RS we still use the stable one, so this is not changing anything we release